### ISSUE TITLE
source activate discards /usr/bin

### ISSUE THREAD

**Issue by orangecms** [conda/conda#2382]
I don't know where to put this now since the activate/deactive commands are moving around, but from what I understood they will be put back into conda (?). I am currently using conda 4.0.5 with conda-env 2.4.5 and the following happens in bash:

``` bash
source activate ai-evaluation
discarding /usr/bin from PATH
prepending /home/shopco/.conda/envs/ai-evaluation/bin to PATH
bash: awk: command not found
bash: dirname: command not found
bash: awk: command not found
```

I am normally using fish, and with the fish script everything seems fine. However, I have some scripts to automate running some apps, which are written in bash and would require this to work. I currently use `conda activate foo` in fish and have the source activate commented out in my bash scripts as a workaround.

My `$PATH` looks like the following when I run `bash`:

``` bash
echo $PATH
/home/shopco/.nvm/versions/node/v5.10.0/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/opt/android-sdk/platform-tools:/opt/android-sdk/tools:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
```

I guess the issue is related to conda being installed globally in the system:

``` bash
conda env list
# conda environments:
#
_build                   /home/shopco/.conda/envs/_build
ai-evaluation            /home/shopco/.conda/envs/ai-evaluation
automation-evaluation     /home/shopco/.conda/envs/automation-evaluation
default                  /home/shopco/.conda/envs/default
sc-ai                    /home/shopco/.conda/envs/sc-ai
root                  *  /usr
```

**Comment by kalefranz** [comment]
This repo is fine to file the issue.  And it's an interesting problem.  Do you have conda installed at `/usr/bin/conda`?

**Comment by orangecms** [comment]
Yes, activate/deactivate are also there.

**Comment by sh4wn** [comment]
I have the same issue with zsh. 

OS: Arch Linux

Installed using these AUR packages:

- https://aur.archlinux.org/packages/python-conda-env/
- https://aur.archlinux.org/packages/python-conda/

I can confirm that conda and the activate script both are installed in /usr/bin.

**Comment by orangecms** [comment]
Cool, I created that package @sh4wn :)

I will see if the issue can be easily resolved. You can run `conda activate` in the fish shell as a workaround. ;)

**Comment by kalefranz** [comment]
That's awesome you have those packages in arch. Will definitely be looking into how we can work better here.

**Comment by kalefranz** [comment]
I'm not sure if this is a bug or a feature request, but I put the bug label on it.  

For some context, in conda's earlier days, I think there was an assumption made that it wouldn't be installed next to a system package manager like pacman or yum or apt-get.  There might even be places throughout the code this assumption might even be enforced with hard sys.exit calls (or maybe they've gone away at this point).  It was easier for conda to assume it wasn't at /usr/bin so it wouldn't have to worry about trampling on top of another package manager, or having another package manager trample on top of it.  We might be at the point where we can start to think through how to do better.  I very much want conda to feel exactly like a completely native package manager everywhere its used.

**Comment by orangecms** [comment]
Well, pip for example is usually installed system-wide as well. In FreeBSD, they recently added a warning that you should never, ever install anything to the system environment through pip, only locally. It's similar with npm (the Node.js package manager), and I'm having the discussion with the developers of fisherman as well (a plugin manager for the fish shell).
It is in general prefarable not to put binaries into the user's home directory, because everyone puts them in different places, they behave weird after time because users might accidentally interfer and delete things etc., so that only creates a mess.
I mainly wrote the packages because I didn't want to execute a bash script with binary data in it like Miniconda or, even worse, Anaconda (~250MB). That just felt strange. Plus, my system packager can take care of the dependencies, keep them up to date, and I can report back to upstream should anything break. So that's also a profit for conda I hope. :)

**Comment by attardi** [comment]
I do have the same problem after installing with:

    sudo -H pip install conda

**Comment by orangecms** [comment]
@attardi which system are you on?

**Comment by attardi** [comment]
ubuntu 14.04

**Comment by orangecms** [comment]
Nice, I'm fighting a bit with the Debian packaging utilities, but I might get a working build soon so you can install conda through apt-get from a PPA repo. We're also running Ubuntu on our servers. ;)
Anyway, let's file a PR on conda maybe that fixes the issue. :)

**Comment by orangecms** [comment]
My current solution is to patch out the line that removes the root env from `$PATH`:
https://github.com/orangecms/conda/commit/67faa8d7b08b2876f155bd9af5e2a0caec19ae2f

However, I get tons of errors when running the tests, even with the current master branch. :/

I'm now monkey-patching the `activate.py` in the PKGBUILD, so you can use it with bash in Arch Linux now @sh4wn - but you need to upgrade `conda-env` as well because the file locations would collide otherwise. The fish script doesn't work with 4.1.x currently.

**Comment by kalefranz** [comment]
If this problem isn't already resolved, it for sure will be with conda 4.4 and #5044

**Comment by orangecms** [comment]
Thanks a lot, it's fine now @kalefranz - I really appreciate the work here, you're providing good quality software that is reproducible and nice to use! :)
